pkg/clustertree/cluster-manager/utils/leaf_model_handler.go

```diff
@@ -159,19 +159,9 @@ func (h ClassificationHandler) UpdateRootNodeStatus(ctx context.Context, nodesIn
 				return err
 			}
 
-			patch, err := utils.CreateMergePatch(nodeInRoot, rootCopy)
-			if err != nil {
-				return fmt.Errorf("failed to CreateMergePatch while update join node %s status, err: %v", nodeNameInRoot, err)
-			}
-
-			if latestNode, err := h.RootClientset.CoreV1().Nodes().PatchStatus(ctx, node.Name, patch); err != nil {
+			rootCopy.Status.Addresses = updateAddress
+			if _, err = h.RootClientset.CoreV1().Nodes().UpdateStatus(ctx, rootCopy, metav1.UpdateOptions{}); err != nil {
```

Comment:
Use `retry` or set RV to 0 to prevent RV conflicts

Reply:
This is already within the ```RetryOnConflict```. line 123 https://github.com/kosmos-io/kosmos/pull/427/files#diff-833c68f55a5ffa527cfb14bbf6b5f0dadb6e7c82bc07bcf95b976a2ab1dfa472R123